lib/teleterm/gateway/config.go

```diff
@@ -69,7 +72,13 @@ func (c *Config) CheckAndSetDefaults() error {
 	}
 
 	if c.LocalAddress == "" {
-		c.LocalAddress = "localhost"
+		// Due to SQL Server Management Studio behavior, it can't connect to address like localhost:12345,
+		// we have to use IP, like 127.0.0.1:12345.
+		if runtime.GOOS == constants.WindowsOS && c.Protocol == defaults.ProtocolSQLServer {
+			c.LocalAddress = "127.0.0.1"
+		} else {
+			c.LocalAddress = "localhost"
+		}
```

Comment:
Can you get rid of the else and default to localhost like you had it originally?

```
c.LocalAddress = "localhost"
if runtime.GOOS == constants.WindowsOS ... {
    c.LocalAddress = "127.0.0.1"
}
```

Reply:
Done